astropy/coordinates/earth.py

```diff
@@ -639,7 +639,7 @@ def get_itrs(self, obstime=None):
         """
         # Broadcast for a single position at multiple times, but don't attempt
         # to be more general here.
-        if obstime and self.size == 1 and obstime.size > 1:
+        if obstime and self.size == 1 and obstime.shape:
```

Comment:
Note: I did not change `self.size == 1` to `self.shape == ()` here as that might lead to further regressions, and I don't think it is all that bad to broadcast a location with, say, shape (1, 1) against a time with (2, 2). (Indeed, at some level I think the broadcasting should always, but that would be for a new feature, not a bug fix).